plugins/woocommerce-blocks/tests/e2e/tests/product-gallery/inner-blocks/product-gallery-large-image-next-previous/product-gallery-large-image-next-previous.block_theme.spec.ts

```diff
@@ -111,7 +102,7 @@ test.describe( `${ blockData.name }`, () => {
 	} );
 
 	// eslint-disable-next-line playwright/no-skipped-test
```

Comment:
Comments can be removed if you unskip the tests. Applies to all the tests.

Reply:
https://github.com/woocommerce/woocommerce/pull/54561/commits/fdbf946540f417aa32351fd44c27a8477dc20d62